### ISSUE TITLE
unable to  create environment using yml file

### ISSUE THREAD

**Issue by Engr-KhurramIqbal** [MingjunZhong/transferNILM#4]
@MingjunZhong 
hello Dr Zhong,I am facing difficulties to create this specific environment on Windows OS.I think the provided yml file is for Linux not for windows.will you please provide some guidelines about the reproduction of results on windows OS. I will be waiting for your positive response,thanks
Kind regards.

**Comment by MingjunZhong** [comment]
@khurramch 
I haven't worked it on Windows yet, and unfortunately I have no plan to make it on Windows recently. Could you try Ubuntu on Windows which is similar when you are working on Linux?

**Comment by khurramch** [comment]
> @khurramch
> I haven't worked it on Windows yet, and unfortunately I have no plan to make it on Windows recently. Could you try Ubuntu on Windows which is similar when you are working on Linux?

**Comment by Engr-KhurramIqbal** [comment]
@MingjunZhong actually I really do not have any experience of using LINUX.Is it possible to get your help while producing the results using ubuntu

**Comment by MingjunZhong** [comment]
@khurramch Yes. I think programming under Linux is not avoidable if you want to do this kind of projects - Data Science and AI. It is always good to learn and won't take long for you to learn Linux if you already have programming skills.

**Comment by Engr-KhurramIqbal** [comment]
@MingjunZhong actually I am trying to reproduce some useful results for the degree requirement of my MSc Engg. I was working on nilmtk and nilmtk-contri for many days but unable to get some results. Then I saw this Repository,it looks good but I am unable to proceed due to variation in OS.Will you pls guide me enough to get some useful results.

**Comment by MingjunZhong** [comment]
@khurramch Hmmm, when using nilmtk-contrib, you still need Linux, don't you? What was the problems when using nilmtk-contrib? I am afraid I won't help much in terms of how to program under Windows.

**Comment by Engr-KhurramIqbal** [comment]
@MingjunZhong I am unable to reproduce results related to their paper work.I open an issue on the nilmtk-contrib repository .You can see [here](https://github.com/nilmtk/nilmtk-contrib/issues/28) .I will try to follow your instruction while working on Ubuntu,I hope you will help me in this journey.
